config/api/src/main/java/org/jboss/windup/config/WindupRuleProvider.java

```diff
@@ -52,30 +63,52 @@ public String getID()
      */
     public Class<? extends RulePhase> getPhase()
     {
-        return DEFAULT_PHASE;
+        Rules rulesAnnotation = this.getClass().getAnnotation(Rules.class);
+        if(rulesAnnotation == null)
+            return DEFAULT_PHASE;
+        if(rulesAnnotation.phase() == null)
+            return DEFAULT_PHASE;
+        return rulesAnnotation.phase();
     }
 
     /**
-     * Specify additional meta-data about the {@link Rule} instances originating from this {@link WindupRuleProvider}.
+     * Specify additional meta-data about the {@link Rule}s instances originating from this {@link WindupRuleProvider}.
```

Comment:
Either the "s" after rules should go away or the word "instances" should go away.